src/Inkluzitron/Data/Entities/DailyUserActivity.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Discord;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Inkluzitron.Data.Entities
+{
+    public class DailyUserActivity
```

Comment:
DailyUserActivity nemá timestamp/row version, takže imho hrozí to samé, co dřív u Usera. Tzn. že se seběhnou dvě vlákna, každé zkusí inkrementovat nějakou z hodnot, a jenom jedno vyhraje.
Mělo by to jít vyřešit přidáním RowVersion/TS stejně jako to má teď User + práci s ukládáním/upravováním instancí zabalit do toho Patiently wrapperu.
Naopak User to teď už možná (?) nepotřebuje.

Reply:
`User` to potřebuje, protože `LastMessagePointsIncrement` a `LastReactionPointsIncrement`